src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/MerchandiseReturnController.php

```diff
@@ -75,6 +83,95 @@ public function indexAction(Request $request, MerchandiseReturnFilters $filters)
         ]);
     }
 
+    /**
+     * Edit existing order return
+     *
+     * @AdminSecurity(
+     *     "is_granted(['update'], request.get('_legacy_controller'))",
+     *     redirectRoute="admin_merchandise_returns_index"
+     * )
+     *
+     * @param int $orderReturnId
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function editAction(int $orderReturnId, Request $request): Response
+    {
+        $formBuilder = $this->get('prestashop.core.form.identifiable_object.builder.order_return_form_builder');
+        $formHandler = $this->get('prestashop.core.form.identifiable_object.handler.order_return_form_handler');
+
+        try {
+            /** @var EditableOrderReturn $editableOrderReturn */
+            $editableOrderReturn = $this->getQueryBus()->handle(
+                new GetOrderReturnForEditing(
+                    $orderReturnId
+                )
+            );
+
+            $form = $formBuilder->getFormFor($orderReturnId);
+            $form->handleRequest($request);
+
+            $result = $formHandler->handleFor($orderReturnId, $form);
+
+            if ($result->getIdentifiableObjectId()) {
```

Comment:
```suggestion
            if ($result->isSubmitted() && $result->isValid()) {
```

We don't care much about the returned ID here since it's an edit action, the most important is that the updated suceeded